js/npm_import.bzl

```diff
@@ -159,3 +190,13 @@ translate_package_lock = repository_rule(
     implementation = lib.implementation,
     attrs = lib.attrs,
 )
+
+def _has_postinstall(repository_ctx, package_path):
```

Comment:
there is also preinstall
https://docs.npmjs.com/cli/v8/using-npm/scripts#life-cycle-scripts
but I don't think we need to cover everything yet